model_analyzer/result/result_manager.py

```diff
@@ -83,23 +85,25 @@ def _init_state(self):
         """
 
         self._state_manager.set_state_variable('ResultManager.results', {})
-        self._state_manager.set_state_variable(
-            'ResultManager.server_only_data', {})
+        self._state_manager.set_state_variable('ResultManager.server_only_data',
+                                               {})
 
     def _create_server_table(self):
         # Server only
         server_output_headers = []
-        server_output_fields = self._config.server_output_fields
-        for server_output_field in server_output_fields:
+        server_output_fields = []
+        for server_output_field in self._config.server_output_fields:
             if server_output_field in self.headers:
                 server_output_headers.append(self.headers[server_output_field])
             elif server_output_field in self._gpu_metrics_to_headers:
                 server_output_headers.append(
                     self._gpu_metrics_to_headers[server_output_field])
             else:
-                raise TritonModelAnalyzerException(
-                    f'Server output field "{server_output_field}", does not exist'
-                )
+                logging.warning(
+                    f'Server output field "{server_output_field}", has no data')
```

Comment:
Is it because the CPU metrics are disabled? If I understood correctly, it is going to be displayed on every run run of MA using default arguments, right?

Reply:
Yes. The `profile` command does not include CPU metrics, but the `analyze` command includes CPU metrics by default. This will cause it to print.

Reply:
Is there any way to prevent this? I think using this warning we are pushing the user toward two different contradictory directions. If they have CPU metrics inside the `--metrics` flag, we are going to display a warning that it affects the performance. If they do not use CPU metrics they are going to receive the warning above when running analyze. I think the MA run using the default configurations should be warning free (unless something has gone wrong).

Reply:
Yes we don't want warnings on default runs. If we are changing the default profile to exclude CPU metrics, the default analyze will also have to exclude them from all the tables and reports.

Reply:
Maybe we can just print an `info` instead of `warning`?